pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -83,6 +95,7 @@ def __init__(
         ddp_comm_state: Optional[object] = None,
         ddp_comm_hook: Optional[callable] = None,
         ddp_comm_wrapper: Optional[callable] = None,
+        model_averaging_period: Optional[int] = None,
```

Comment:
Why not put 16 as a default right there ?

Reply:
For most cases `model_averaging_period` will be not used at all, so personally prefer a default `None` to avoid any confusion from the user. This way, if post-localSDG is not used, the user does not need to care about the effect of this arg.